ytfzf

```diff
@@ -1063,7 +1064,11 @@ while :; do
 
 	[ $enable_hist -eq 1 ] && add_to_hist "$ytfzf_video_json_file" < "$ytfzf_selected_urls"
 
-	format_urls < "$ytfzf_selected_urls" | open_player
+	if [ $force_yt -eq 1 ]; then
```

Comment:
force_yt needs to be declared somewhere where the rest of the variables are declared or this will be an error when `--force-youtube` isn't given.